### PR TITLE
test: Failing tests for `create_token_pool` instruction

### PR DESCRIPTION
**Pull request by vadorovsky**

### CODE REVIEW

File: test-programs/compressed-token-test/tests/test.rs [hunk 1]
```diff
@@ -43,6 +49,108 @@ async fn test_create_mint() {
...
+            cpi_authority_pda: get_cpi_authority_pda().0,
+        };
+        let instruction = Instruction {
+            program_id: light_compressed_token::ID,
+            accounts: accounts.to_account_metas(Some(true)),
+            data: instruction_data.data(),
+        };
+        rpc.create_and_send_transaction(&[instruction], &payer.pubkey(), &[&payer])
+            .await
+            .unwrap_err();
```
Comment (ananas-block) [inline test-programs/compressed-token-test/tests/test.rs:150]: can we assert the explicit error here?

**Comment by ananas-block** [comment]
also ci wasn't triggered properly

**Comment by ananas-block** [comment]
meant a custom error code, `ErrorCode::ConstraintSeeds` is not that specific.

**Comment by vadorovsky** [comment]
> meant a custom error code, `ErrorCode::ConstraintSeeds` is not that specific.

There is no support for custom errors with `@` in `seeds` in Anchor, see the comment from the other PR

https://github.com/Lightprotocol/light-protocol/pull/964#discussion_r1670007259

**Review by ananas-block — APPROVED** [review]

**Pull request merged**